akka-projection-core/src/main/scala/akka/projection/scaladsl/ProjectionHandler.scala

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (C) 2019 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.projection.scaladsl
+
+import akka.Done
+
+import scala.concurrent.{ExecutionContext, Future}
+import scala.language.higherKinds
+import scala.util.control.NonFatal
+
+trait ProjectionHandler[Event, IO] {
```

Comment:
Renamed it to `ProjectionHandler` to align with Akka Persistence Typed naming. The user function is the handler of the incoming event. The `Projection` is the whole that runs and manage the offset, cluster distribution, restarts, etc.